### PR TITLE
feat(AniLibria): add presence

### PR DESCRIPTION
**Pull request by FlimixST**
## Description 
Added presence for [AniLibria](https://anilibria.top/).

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

[Screenshots](https://imgur.com/a/ToYqjdl)

idk, I couldn't directly paste the images into github.

</details>

### CODE REVIEW

**Comment by FlimixST** [comment]
very funny error in metadata was, forgot to change the site address in the description, because the description was taken from another project xD

File: websites/A/AniLibria/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	"$schema": "https://schemas.premid.app/metadata/1.11",
+	"apiVersion": 1,
+	"author": {
+		"id": "1120534568018116669",
+		"name": "flimixofficial"
+	},
+	"service": "AniLibria",
+	"description": {
+		"en": "AniLibria.top is Russian anime website",
+		"ru": "Jut.su — это русский сайт с аниме"
```
Comment (FlimixST) [inline websites/A/AniLibria/metadata.json]: xD

File: websites/A/AniLibria/metadata.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+		"name": "flimixofficial"
+	},
+	"service": "AniLibria",
+	"description": {
+		"en": "AniLibria.top is Russian anime website",
+		"ru": "AniLibria.top — это русский сайт с аниме"
+	},
+	"url": "anilibria.top",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/HrvpNEc.png",
```
Comment (theusaf) [inline websites/A/AniLibria/metadata.json:15]: Logo must be 512x512px

File: websites/A/AniLibria/presence.ts [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+	return cachedTime;
+}
+
+presence.on("UpdateData", async () => {
+	const { pathname, href } = document.location,
+		ogTitle = document
+			.querySelector('meta[property="og:title"]')
+			?.getAttribute("content"),
+		video = document.querySelector("video"),
+		PATHS = {
```
Comment (theusaf) [inline websites/A/AniLibria/presence.ts:16]: Consider placing this variable outside of the main loop for readability and performance

File: websites/A/AniLibria/presence.ts [hunk 4]
```diff
@@ -0,0 +1,133 @@
+const presence = new Presence({ clientId: "1165759293576982578" });
+
+let cachedTime = 0;
+
+export function adjustTimeError(time: number, acceptableError: number): number {
+	if (Math.abs(time - cachedTime) > acceptableError) cachedTime = time;
+	return cachedTime;
+}
```
Comment (theusaf) [inline websites/A/AniLibria/presence.ts:8]: This function may be unnecessary

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/A/AniLibria/presence.ts [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+		if (match && match[2]) {
+			episodeNumber = match[1];
+			releaseTitle = match[1];
+			animeTitle = match[3] || match[2];
+		} else releaseTitle = ogTitle;
+	}
+
+	const presenceData: PresenceData = {
+		type: ActivityType.Watching,
+	};
```
Comment (theusaf) [inline websites/A/AniLibria/presence.ts:44]:
```suggestion
	const presenceData: PresenceData = {
		type: ActivityType.Watching,
		largeImageKey: "https://i.imgur.com/shUaioa.png",
	};
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**